_tools/generateFontData.js

```diff
@@ -96,6 +91,14 @@ const findFontFile = async directory => {
 	return paths;
 };
 
+const getMetaData = async (fontData, fontFile) => {
```

Comment:
Should this really be an async function? It doesn't seem to do anything async.

Reply:
Good call, I was inspired by all the surrounding async stuff, will fix!